Headlines/src/Controllers/NewsTableViewController.swift

```diff
@@ -10,36 +10,35 @@ import UIKit
 import SafariServices
 import Crashlytics
 
-class NewsTableViewController: UITableViewController, NewsCellViewModelDelegate {
-
+class NewsTableViewController: UITableViewController,
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/c44c4aee7ab76439e79aef49b1bea44832708f73/Headlines/src/Controllers/NewsTableViewController.swift#L13"></span>Type body should span 200 lines or less excluding comments and whitespace: currently spans 235 lines</td>
    </tr>
  </tbody>
</table>